app/lib/forms/helpers/institution.rb

```diff
@@ -3,36 +3,36 @@ module Helpers
     module Institution
     private
 
-      def institution(source: wizard.store["institution_identifier"])
+      def institution(source: wizard.store["institution_identifier"], application: nil)
         return @institution if @institution
         return nil if source.nil?
 
         klass, identifier = source.split("-")
-
+        store = application.nil? ? query_store : application
```

Comment:
I would rename `store` to `application` as later we are using methods which sounds better eg: `application.works_in_childcare?`.